src/Controls/src/Core/Handlers/Items/iOS/StructuredItemsViewController.cs

```diff
@@ -46,6 +46,16 @@ protected override void Dispose(bool disposing)
 					_footerViewFormsElement.MeasureInvalidated -= OnFormsElementMeasureInvalidated;
 				}
 
+				if (_headerUIView is MauiView hv)
+				{
+					hv.LayoutChanged += HeaderView_LayoutChanged;
```

Comment:
What if you subscribe to SizeChanged on the header/footer xplat elements?

Reply:
When I try to do the following: 
`_headerViewFormsElement.SizeChanged += _headerViewFormsElement_SizeChanged;`

`_headerViewFormsElement_SizeChanged` never triggers